sycl/test-e2e/Basic/image/srgba-read.cpp

```diff
@@ -2,9 +2,8 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
 // RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
 
-// XFAIL: level_zero
-// UNSUPPORTED: cuda
-// UNSUPPORTED: hip
+// Only opencl plugin has implemented the supoort.
+// REQUIRES: opencl
```

Comment:
Now that the aspect check works, should we maybe just do the check and an early exit instead? In the future we would that to a `REQUIRES: aspect_image && aspect_ext_oneapi_srgb` or something to that effect.

Reply:
The purpose of this test is FileCheck which we can't nicely implement without dynamic LIT features support.

Reply:
Oh, I see. Good point. Should we have a TODO just to remind us?